### ISSUE TITLE
Package.activateResources is deprecated.

### ISSUE THREAD

**Issue by hkdnet** [TabDigital/atom-mocha-test-runner#30]
The context menu CSON format has changed. Please see
https://atom.io/docs/api/latest/ContextMenuManager

**Comment by marcopiraccini** [comment]
Indeed, I've created a PR for that: https://github.com/TabDigital/atom-mocha-test-runner/pull/27

**Comment by hkdnet** [comment]
I'm sorry not to notice that PR
Thanks a lot :+1:

**Comment by rprieto** [comment]
Hi @hkdnet, we just released version `0.4.1`, can you check if it works now?

**Comment by nguyenchr** [comment]
@rprieto confirmed that the context menu is working for me
